app/javascript/ui/casebooks/dragging.coffee

```diff
@@ -0,0 +1,97 @@
+ui = require 'ui/turbolinks'
```

Comment:
I had https://github.com/rails/webpacker w/ sprockets-es6  recommended to me. 

but if you want to do yarn that's fine with me